tests/cpp/tree/test_gpu_hist.cu

```diff
@@ -162,81 +162,6 @@ inline GradientQuantizer DummyRoundingFactor() {
   return GradientQuantizer(dh::ToSpan(gpair));
 }
 
-// TODO(trivialfis): This test is over simplified.
-TEST(GpuHist, EvaluateRootSplit) {
```

Comment:
I removed this test because it specifies a dense dataset but then also has missing gradients, which now triggers a run time check. It was easier to remove it than rewrite it.